xmppcat/__main__.py

```diff
@@ -61,13 +61,15 @@ def main():
         message = sys.stdin.read()
     pwd = args.pwd if args.pwd else config.get('pass', '')
     user = args.user if args.user else config.get('user', '')
+    host = args.host if args.host else config.get('host', '')
+    port = args.port if args.port else config.get('port', '')
```

Comment:
args.port always evaluates to True here since the argument has a default value of `int(5222)`.
Also the value, when taken from configparser, should be cast to int.